lib/elasticity/index_config.rb

```diff
@@ -60,5 +63,12 @@ def validate!
         raise "#{attr} is not set" if public_send(attr).nil?
       end
     end
+
+    def settings
```

Comment:
now that i'm thinking about it, maybe it would make more sense that you can arbitrarily set settings hash in the config
```
configure do |c|
  c.settings = { number_of_shards: 1 }
  ...
end
```

then we can make the settings method more simple and universal so we don't have to modify elasticity every time we think of another settings variable we want to customize

```
def settings
  @elasticity_config.settings.merge(@settings || {})
end
```
Something along those lines.  Thoughts?